lib/commons/color/color.js

```diff
@@ -1,6 +1,7 @@
-import { Colorjs } from '../../core/imports';
+import { Colorjs, ArrayFrom } from '../../core/imports';
 
 const hexRegex = /^#[0-9a-f]{3,8}$/i;
+const hslRegex = /hsl\(\s*([\d.]+)(rad|turn)/;
```

Comment:
This fails the following missing unit tests:

js
```
      it('supports negative rad on hue', () => {
        const c = new Color();
        c.parseColorFnString('hsl(-3.49rad, 40%, 50%)');
        assert.equal(c.red, 77);
        assert.equal(c.green, 179);
        assert.equal(c.blue, 144);
        assert.equal(c.alpha, 1);
      });

      it('supports negative turn on hue', () => {
        const c = new Color();
        c.parseColorFnString('hsl(-0.556turn, 40%, 50%)');
        assert.equal(c.red, 77);
        assert.equal(c.green, 179);
        assert.equal(c.blue, 144);
        assert.equal(c.alpha, 1);
      });
```

Reply:
Good catch!